DataFormats/HcalDetId/src/HcalCalibDetId.cc

```diff
@@ -33,6 +33,18 @@ HcalCalibDetId::HcalCalibDetId(int ieta, int iphi) : HcalOtherDetId(HcalCalibrat
       |(((ieta > 0)?(1):(0))<<11); // z side, bit [11]
 }
 
+HcalCalibDetId::HcalCalibDetId(CalibDetType dt, int value) : HcalOtherDetId(HcalCalibration) {
+  id_|=(dt<<17);
+  id_|=value&0xFF;
```

Comment:
Yes.  The relatively generic "value" argument is used because there can only be one constructor for the cases where an single integer is needed along with the CalibDetType and that value might have a different meaning depending on the CalibDetType.